tests/instance_validations_test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+require('./helper');
+
+const chai = require('chai'),
+  expect = chai.expect,
+  Sequelize = require('../source');
+
+describe('InstanceValidator', () => {
+  describe('#update', () => {
+    // Edited test. Changed findByPk parameter.
+    it('should allow us to update specific columns without tripping the validations', async function() {
+      const User = this.sequelize.define('model', {
+        username: Sequelize.STRING,
+        email: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          validate: {
+            isEmail: {
+              msg: 'You must enter a valid email address'
+            }
+          }
+        }
+      });
+
+      await User.sync({ force: true });
+      const user = await User.create({ username: 'bob', email: 'hello@world.com' });
+
+      await User
+        .update({ username: 'toni' }, { where: { id: user.id } });
+
+      // Edited PK. It was 1, now it is dynamically obtained.
+      const user0 = await User.findByPk(user.id);
+      expect(user0.username).to.equal('toni');
+    });
+
+    // Reason: Errors array need DB-level details to be generated. Since it doesn't,
```

Comment:
Do we have an issue open about this?

If not, please open one and add link the issue instead. LGTM after that

Reply:
We do. Will look for the Issue and I'll edit here.